plugins/AIActivity/Controller.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\AIActivity;
+
+use Piwik\NoAccessException;
+use Piwik\Piwik;
+use Piwik\Plugins\AIActivity\FeatureFlags\AIActivity as AIActivityFeatureFlag;
+use Piwik\Plugins\FeatureFlags\FeatureFlagManager;
+use Piwik\Request;
+
+class Controller extends \Piwik\Plugin\Controller
+{
+    /**
+     * @var FeatureFlagManager
+     */
+    private $featureFlagManager;
+
+    public function __construct(FeatureFlagManager $featureFlagManager)
+    {
+        parent::__construct();
+
+        $this->featureFlagManager = $featureFlagManager;
+    }
+
+    public function getEvolutionGraph(): string
+    {
+        $this->checkSitePermission();
+
+        if (false === $this->featureFlagManager->isFeatureActive(AIActivityFeatureFlag::class)) {
+            throw new NoAccessException('Feature not available');
+        }
+
+        $view = $this->getLastUnitGraph(
+            $this->pluginName,
+            __FUNCTION__,
+            'AIActivity.get'
+        );
+
+        $view->config->selectable_columns = [
+            Metrics::METRIC_NB_VISITS_AI_AGENTS,
```

Comment:
Wondering if it could make sense to also have the "normal" metrics available here, so it's easy to compare them directly in the chart.